db/db_impl_compaction_flush.cc

```diff
@@ -344,6 +344,18 @@ Status DBImpl::AtomicFlushMemTablesToOutputFiles(
         break;
       }
     }
+    if (num_cfs > 1) {
+      TEST_SYNC_POINT(
+          "DBImpl::AtomicFlushMemTablesToOutputFiles:SomeFlushJobsComplete:1");
+      TEST_SYNC_POINT(
+          "DBImpl::AtomicFlushMemTablesToOutputFiles:SomeFlushJobsComplete:2");
+    }
+    exec_status[0].second =
```

Comment:
do we need to do this even when `s` is non-ok?

also it's not so obvious to me why we run jobs in the order 1, ..., `num_cfs-1`, 0.

Reply:
> do we need to do this even when `s` is non-ok?
What are you referring to by 'this'?
>  also it's not so obvious to me why we run jobs in the order
In future, if we want to parallelize the flush jobs, then this current thread runs job[0], while other num_cfs-1 threads run jobs 1, ...num_cfs-1

Reply:
Discussed offline, we should check `s` before running `jobs[0]`, at least for now.